README.md

```diff
@@ -59,13 +59,15 @@ Alternatively, you can configure username and password in the `sonatype` block:
 nexusPublishing {
     repositories {
         sonatype {
-            username = "your-username"
-            password = "your-password"
+            username = "your-username"    //preferably your API username
+            password = "your-password"    //preferably your API key
```

Comment:
What do you think if API key is the only suggested value here?
What if `password` was renamed like `apiKey` and `password` field was deprecated (e.g. deprecated=hidden)?

Reply:
I like the idea of using `api-username` and `api-key` as the suggested values here.

Changing fields is somehow more tricky as `username` is not the same as `api-username`. In addition, Gradle plugin portal used another convention with `key` and `secret`. They can confuse users. The majority of Sonatype/Nexus users don't know anything about the API key/token. We would probably have to add both the new fields. I will create a separate issue to gather feedback about it.

Reply:
I changed that section a little bit. @vlsi WDYT?

Reply:
Nice, thanks you

Reply:
One more change, I checked the Nexus naming convention and rephrased API username and password to user token's username and password. I'm not sure how to name it in the least confusing way.